helios-client/src/main/java/com/spotify/helios/common/descriptors/Job.java

```diff
@@ -212,6 +223,8 @@ public Job(@JsonProperty("id") final JobId id,
     this.securityOpt = Optional.fromNullable(securityOpt).or(EMPTY_SECURITY_OPT);
     this.networkMode = Optional.fromNullable(networkMode).orNull();
     this.metadata = Optional.fromNullable(metadata).or(EMPTY_METADATA);
+    this.capAdds = Optional.fromNullable(capAdds).or(EMPTY_CAPS);
+    this.capDrops = Optional.fromNullable(capDrops).or(EMPTY_CAPS);
```

Comment:
I interpret it more as don't bother creating an Optional just to use `or(..)` when `firstNonNull(a, b)` is clearer